### ISSUE TITLE
why the transfer limit is only for last 30 days?

### ISSUE THREAD

**Issue by Simplici** [Jigsaw-Code/outline-server#1265]
Hello,

I have a question concerning the transfer limit on key.
It seems that the transfer limit is for the last 30 days instead of forever. 
Is it possible to set a transfer limit forever instead of for last 30 days?

**Comment by bemasc** [comment]
No, this feature is not currently supported.  That behavior would make access keys permanently unusable once the limit is reached, so it seems difficult for administrators to manage.

Could you explain more about why you want this behavior?

**Comment by Simplici** [comment]
from my point of view, I want to control the total transfer used instead of transfer used for last 30 days. For example, I will give every one a key with 1000G transfer, if you have used all of transfer, the user should apply for a new one, the old key is deprecated.
